Sources/SwiftProtobuf/TextDecoder.swift

```diff
@@ -0,0 +1,211 @@
+// ProtobufRuntime/Sources/Protobuf/ProtobufTextDecoding.swift - Text format decoding
```

Comment:
I'm planning soon to do a semi-automated pass to fix all of these up across the project.